config/locales/sv.yml

```diff
@@ -117,6 +117,13 @@ sv:
       unsubscribe: Säga upp
       username: Användarnamn
       web: Webb
+    account_moderation_notes:
```

Comment:
duplicate of [line 53](https://github.com/tootsuite/mastodon/pull/5441/files#diff-3c1154d73d9ad84a41fa6fcc6a890981L53)